src/cloud-code/httpRequest.js

```diff
@@ -45,11 +57,11 @@ module.exports = function(options) {
     var httpResponse = {};
     httpResponse.status = response.statusCode;
     httpResponse.headers = response.headers;
-    httpResponse.buffer = new Buffer(response.body);
+    httpResponse.buffer = response.body;
     httpResponse.cookies = response.headers["set-cookie"];
-    httpResponse.text = response.body;
+    httpResponse.text = response.body.toString('utf-8');
```

Comment:
Shouldn't this be in the `try` block? If the response is not a string or if it fails to be converted to a string - will it throw?